lib/PuppeteerSharp.Tests/Page/EmulateTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class EmulateTests : PuppeteerPageBaseTest
+    {
+        [Fact]
+        public async Task ShouldWork()
+        {
+            var iPhone = DeviceDescriptors.Get(DeviceDescriptorName.IPhone6);
+
+            await Page.GoToAsync(TestConstants.ServerUrl + "/mobile.html");
+            await Page.EmulateAsync(iPhone);
+
+            Assert.Equal(375, await Page.EvaluateFunctionAsync<int>("() => window.innerWidth"));
```

Comment:
I'm not asking to change this, but Unit tests are not only helpful to test the app but also as a "How to" documentation.

I would like to take the community to use functions only when needed. This could be solved as an expression
 `await Page.EvaluateExpressionAsync<int>("window.innerWidth")`. I think the code would be cleaner in this way.

Reply:
right.
I just copied the way puppeteer did it...

Reply:
Yeah I know.